README_KR.md

```diff
@@ -0,0 +1,110 @@
+# euphony-listener
+​
+## 소개
+​
+Euphony-Listener는 Speaker로부터 음파를 통해 데이터를 받아 인식하는 기능을 합니다. \
+스피커가 데이터를 보낼 때 주파수가 발생하게 되는데, Listen 버튼을 누르면 Listener가 데이터를 인식하여 그 결과를 확인할 수 있습니다. \
+android에서만 동작을 합니다.
+​
+## Euphony 라이브러리
+​
+해당 음파 데이터 통신 라이브러리, Euphony의 버전은 안드로이드를 위한 버전입니다.\
+Euphony는 마이크와 녹음기를 활용하여 다른 단말기(안드로이드, 웹)들과 통신하기 위해 디자인 된 무선 라이브러리를 제공합니다. 
+​
+Official Facebook Page : https://www.facebook.com/euphonyproject\
+Official Library Site : https://dev.jbear.co/euphony\
+공식 페이스북 페이지 : https://www.facebook.com/euphonyproject\
+공식 라이브러리 주소 : https://dev.jbear.co/euphony
+​
+## 전제 조건
+​
+**build.gradle**
+```java
+dependencies {
+    implementation 'euphony.lib:euphony:0.7.1.6'
+}
+```
+​
+**AndroidManifest.xml**
+```xml
+<uses-permission android:name="android.permission.RECORD_AUDIO" />
+```
+​
+## 빠른 시작
+​
+### 1. 저장소 클론
+​
+```bash
+$ git clone https://github.com/euphony-io/euphony-listener.git
+```
+![clone](https://user-images.githubusercontent.com/47289893/128968369-e30bfc36-3c57-418d-b3b2-b8976436493b.png)
+​
+### 2. 안드로이드 스튜디오에서 열기
+​
+​
+clone한 프로젝트의 경로를 찾아서 열어준다.
+​
+![open](https://user-images.githubusercontent.com/47289893/128968509-21778091-1c48-432d-8c68-856d89c59a07.png)
+​
+Trust Project 버튼을 누른다.
+​
+![trust](https://user-images.githubusercontent.com/47289893/128968544-78756386-1740-43e7-9f27-78f9322307f8.png)
+​
+### 3. JDK 설정
+​
+처음 열 때 JDK location error가 발생할 수 있다.
+​
+![set jdk-1](https://user-images.githubusercontent.com/47289893/128968605-e73af820-0ae7-4e8c-997a-1c1cdbef7129.png)
+​
+Change JDK location을 클릭하여 Gradle JDK를 지정한다.
+​
+![set jdk-2](https://user-images.githubusercontent.com/47289893/128968614-0988ac95-1672-411d-8e9e-336503be69cd.png)
+​
+프로젝트를 다시 열면 build가 시작된다.
+​
+![build](https://user-images.githubusercontent.com/47289893/128968741-00b1cd80-a9f7-4481-bd1a-ff388141080f.png)
+​
+### 4. 디바이스 연결
+​
+기본적으로 디바이스가 없다.
+​
+![no device](https://user-images.githubusercontent.com/47289893/128968845-d0868890-cb57-4721-a956-857b871e7393.png)
+​
+개발자 모드가 활성화된 디바이스를 연결하면 자동으로 인식된다.
+​
+![yes device](https://user-images.githubusercontent.com/47289893/128968864-35b7ae72-6fb8-4bb2-8706-1ec2257faf73.png)
+​
+### 5. 테스트
+​
+연결된 디바이스에서 euphony-listener를 동작시킨다.
+![run](https://user-images.githubusercontent.com/47289893/128968893-cca8c520-4dcc-41e7-9e04-9d4849143176.png)
+![run-build](https://user-images.githubusercontent.com/47289893/128969518-043e50ae-aa45-4d0a-b145-9e7d1176353c.png)
```

Comment:
Images and descriptions are not aligned properly. Please align the same as the English version.